porcupine/plugins/run/no_terminal.py

```diff
@@ -83,11 +135,22 @@ def _queue_clearer(self) -> None:
                 if tag == "clear":
                     assert not text
                     self.textwidget.delete("1.0", "end")
+                    for tag in self._links.keys():
+                        self.textwidget.tag_delete(tag)
```

Comment:
Not really necessary, the link-specific tags don't have any bindings or configuration associated with them

```suggestion
```